packages/flutter/lib/src/cupertino/tab_scaffold.dart

```diff
@@ -199,8 +199,8 @@ class _CupertinoTabScaffoldState extends State<CupertinoTabScaffold> {
 
 /// An widget laying out multiple tabs with only one active tab being built
```

Comment:
I know this isn't you comment, but change "An widget" to "A widget".

Reply:
Done. Thanks.